<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Szyfrator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="szyfruj">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Szyfruj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deszyfruj">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Deszyfruj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rozpoznajTwarz">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Zaloguj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dodajUzytkownika">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Dodaj u≈ºytkownika</string>
    </property>
   </widget>
   <widget class="QLabel" name="nazwaUzytkownika">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Zaloguj sie lub utworz swoje konto</string>
    </property>
   </widget>
   <widget class="QLabel" name="obrazKamery">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="wyloguj">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyloguj</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
